geotrigger.js

```diff
@@ -222,7 +222,7 @@
     // create the url for the request
     var url = (geotriggersRequest) ? this.geotriggersUrl + method : method;
 
-    if (this.proxy) {
+    if (this.proxy && !CORS) {
       url = this.proxy + url;
     }
```

Comment:
@jgravois is there a scenario where CORS is supported but you'd still want a proxy to be used?